packages/engine/src/scene/functions/loaders/AmbientLightFunctions.ts

```diff
@@ -26,25 +25,23 @@ export const deserializeAmbientLight: ComponentDeserializeFunction = (
   json: ComponentJson<AmbientLightComponentType>
 ) => {
   const light = new AmbientLight()
+  const props = parseAmbientLightProperties(json.props)
 
   addComponent(entity, Object3DComponent, { value: light })
   addComponent(entity, DisableTransformTagComponent, {})
-  addComponent(entity, AmbientLightComponent, {
-    ...json.props,
-    color: new Color(json.props.color)
-  })
+  addComponent(entity, AmbientLightComponent, props)
 
   if (Engine.isEditor) getComponent(entity, EntityNodeComponent)?.components.push(SCENE_COMPONENT_AMBIENT_LIGHT)
 
-  updateAmbientLight(entity)
+  updateAmbientLight(entity, props)
 }
 
-export const updateAmbientLight: ComponentUpdateFunction = (entity: Entity) => {
+export const updateAmbientLight: ComponentUpdateFunction = (entity: Entity, properties: AmbientLightComponentType) => {
   const component = getComponent(entity, AmbientLightComponent)
   const light = getComponent(entity, Object3DComponent)?.value as AmbientLight
 
-  light.color = component.color
-  light.intensity = component.intensity
+  if (Object.hasOwnProperty.call(properties, 'color')) light.color = component.color
```

Comment:
is there a reason why we're using ES5 syntax instead of ESM?  I prefer `if (typeof properties.color !== 'undefined')`